lib/ui/pointer.dart

```diff
@@ -137,6 +137,9 @@ enum PointerSignalKind {
   unknown
 }
 
+/// A function that implements the [PointerData.respond] method.
+typedef OnPointerDataRespondCallback = void Function({bool allowPlatformDefault});
```

Comment:
nit: drop the "On" from the typedef name (but keep it for the property `onRespond`):

```suggestion
typedef PointerDataRespondCallback = void Function({bool allowPlatformDefault});
```